package.json

```diff
@@ -24,7 +24,9 @@
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
-    "mysql2": "^2.2.5"
+    "multer": "^1.4.2",
+    "mysql2": "^2.2.5",
+    "nodemon": "^2.0.7"
```

Comment:
nodemon should be in dev dependencies: uninstall it, and reinstall it with --save-dev

Reply:
fixed